Error: The `form` element was not requested.
Error: A field group must have an accessible label. Use `fieldset` and `legend`.
Error: For broadest accessibility, implicitly associated `label` elements should also be explicitly associated with their control, using `for`. Note: If radio inputs are wrapped in their `label`, `br` is unnecessary.
diff --git a/output/Radio Group/2/b1fd01c4-2120-49d7-bbf8-a4b469b72c98.html b/output/Radio Group/2/b1fd01c4-2120-49d7-bbf8-a4b469b72c98.html
index 9b44750..aac94c5 100644
--- a/output/Radio Group/2/b1fd01c4-2120-49d7-bbf8-a4b469b72c98.html	
+++ b/output/Radio Group/2/b1fd01c4-2120-49d7-bbf8-a4b469b72c98.html	
@@ -1,13 +1,15 @@
-<form>
-  <label>
-    <input type="radio" name="theme" value="light" checked> Light
+<fieldset>
+  <legend>Choose a theme:</legend>
+  <label for="light">
+    <input type="radio" name="theme" id="light" value="light">
+    Light
   </label>
-  <br>
-  <label>
-    <input type="radio" name="theme" value="dark"> Dark
+  <label for="dark">
+    <input type="radio" name="theme" id="dark" value="dark">
+    Dark
   </label>
-  <br>
-  <label>
-    <input type="radio" name="theme" value="high-contrast"> High Contrast
+  <label for="high-contrast">
+    <input type="radio" name="theme" id="high-contrast" value="high-contrast">
+    High Contrast
   </label>
-</form>
\ No newline at end of file
+</fieldset>
\ No newline at end of file
